src/main/java/com/royalrangers/controller/DropboxController.java

```diff
@@ -1,37 +1,30 @@
 package com.royalrangers.controller;
 
 import com.dropbox.core.DbxException;
-import com.dropbox.core.DbxRequestConfig;
-import com.dropbox.core.v2.DbxClientV2;
 import com.dropbox.core.v2.files.FileMetadata;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.web.bind.annotation.*;
+import com.royalrangers.service.DropboxService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.multipart.MultipartFile;
 
 import java.io.*;
-import java.util.Locale;
 
 @RestController
-@Slf4j
 public class DropboxController {
-
-    @Value("${dropbox.appName}")
-    private String appName;
-    @Value("${dropbox.accessToken}")
-    private String accessToken;
+    @Autowired
+    DropboxService dropboxService;
 
     @PostMapping("/upload")
     public String fileUpload(@RequestParam("file") MultipartFile file) throws IOException, DbxException {
-         String UPLOADED_FOLDER = "C://Users//Public//Pictures//Sample Pictures//";
 
-        DbxRequestConfig config = new DbxRequestConfig(appName, Locale.getDefault().toString());
-        DbxClientV2 client = new DbxClientV2(config, accessToken);
         if (file.isEmpty()) {
             return "Please select a file to upload";
         }
-        InputStream in = new FileInputStream(UPLOADED_FOLDER + file.getOriginalFilename());
-        FileMetadata metadata = client.files().uploadBuilder("/" + file.getOriginalFilename())
+        byte[] bytes = file.getBytes();
+        InputStream in = new ByteArrayInputStream(bytes);
+        FileMetadata metadata = dropboxService.getClient().files().uploadBuilder("/" + file.getOriginalFilename())
                 .uploadAndFinish(in);
```

Comment:
Please move all logic for upload file to Dropbox to DropboxService. In controller you just need to invoke appropriate method from this service with input parameters. In your case with file which you get from UI.